.gitignore

```diff
@@ -0,0 +1,5 @@
+**/__pycache__/*
```

Comment:
You don't need __pycache__ in the remote repository. Uncomment it and remove already pushed files.